src/pages/groupComparison/Overlap.tsx

```diff
@@ -39,11 +41,38 @@ enum PlotType {
 export default class Overlap extends React.Component<IOverlapProps, {}> {
 
     @observable plotExists = false;
+    @observable vennFailed = false;
+
+    private reactionDisposers:IReactionDisposer[] = [];
+
+    constructor(props:IOverlapProps) {
+        super(props);
+
+        this.reactionDisposers.push(
+            // Only when the selected groups change, reset the venn failure marker, to mark that venn hasn't tried yet.
+            reaction(
```

Comment:
hey, should have asked this earlier, but could you not just put a key on the component which reinstantiate it.  i'm just trying to avoid reactions as much as possible because I think they complicate the flow

Reply:
oh okay, sure I'll try that